Source/WebCore/css/SelectorPseudoElementTypeMap.in

```diff
@@ -22,3 +22,5 @@ placeholder, PseudoElementWebKitCustom
 -webkit-scrollbar-track-piece
 selection
 slotted
+thumb, PseudoElementWebKitCustom
+track, PseudoElementWebKitCustom
```

Comment:
I would be happy to re-order this file lexicographically. Seems a bit random at the moment.